<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="ADS67" name="Test" tests="14" failures="5" timestamp="8 Oct 2017 12:55:04 GMT" time="1.534" errors="0">
  <testcase name="ArthaDB_ProdCodes_Check_TC001" time="0.04" classname="exe.Arthadb_Automation"/>
  <testcase name="ArthaDB_AuditDates_Check_TC002" time="0.019" classname="exe.Arthadb_Automation"/>
  <testcase name="ArthaDB_Account_Number_Check_TC003" time="0.005" classname="exe.Arthadb_Automation"/>
  <testcase name="ArthaDB_ChargeType_Check_TC004" time="0.015" classname="exe.Arthadb_Automation"/>
  <testcase name="ArthaDB_Invoice_Number_Check_TC005" time="0.005" classname="exe.Arthadb_Automation"/>
  <testcase name="ArthaDB_Transaction_Amount_Check_TC006" time="0.005" classname="exe.Arthadb_Automation"/>
  <testcase name="ArthaDB_ChargeSign_Check_TC007" time="0.007" classname="exe.Arthadb_Automation">
    <failure type="junit.framework.AssertionFailedError" message="PAYMENTS-Charge Sign Codes check is Failed at Account_Number=ACCD101N8776123, expected:&amp;lt;0&amp;gt; but was:&amp;lt;1&amp;gt;">
      <![CDATA[junit.framework.AssertionFailedError: PAYMENTS-Charge Sign Codes check is Failed at Account_Number=ACCD101N8776123, expected:<0> but was:<1>
	at junit.framework.Assert.fail(Assert.java:57)
	at junit.framework.Assert.failNotEquals(Assert.java:329)
	at junit.framework.Assert.assertEquals(Assert.java:78)
	at junit.framework.Assert.assertEquals(Assert.java:234)
	at exe.Arthadb_Automation.ArthaDB_ChargeSign_Check_TC007(Arthadb_Automation.java:300)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- ArthaDB_ChargeSign_Check_TC007 -->
  <testcase name="ArthaDB_Record_Type_Check_TC008" time="0.005" classname="exe.Arthadb_Automation">
    <failure type="junit.framework.AssertionFailedError" message="PAYMENTS-Record Type check is Failed at Invoice_Number=J9908844423C102,J9160067066C122,J91606706e6C122, expected:&amp;lt;0&amp;gt; but was:&amp;lt;3&amp;gt;">
      <![CDATA[junit.framework.AssertionFailedError: PAYMENTS-Record Type check is Failed at Invoice_Number=J9908844423C102,J9160067066C122,J91606706e6C122, expected:<0> but was:<3>
	at junit.framework.Assert.fail(Assert.java:57)
	at junit.framework.Assert.failNotEquals(Assert.java:329)
	at junit.framework.Assert.assertEquals(Assert.java:78)
	at junit.framework.Assert.assertEquals(Assert.java:234)
	at exe.Arthadb_Automation.ArthaDB_Record_Type_Check_TC008(Arthadb_Automation.java:336)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- ArthaDB_Record_Type_Check_TC008 -->
  <testcase name="ArthaDB_Record_Type_Check_TC009" time="0.006" classname="exe.Arthadb_Automation"/>
  <testcase name="ArthaDB_Premise_City_Check_TC010" time="0.015" classname="exe.Arthadb_Automation"/>
  <testcase name="ArthaDB_Customer_First_Last_Name_Check_TC011" time="0.035" classname="exe.Arthadb_Automation">
    <failure type="junit.framework.AssertionFailedError" message="CUSTOMERS-First &amp;amp; Last Name check is Failed at SSN=123-45-6791,123-45-6792, expected:&amp;lt;0&amp;gt; but was:&amp;lt;2&amp;gt;">
      <![CDATA[junit.framework.AssertionFailedError: CUSTOMERS-First & Last Name check is Failed at SSN=123-45-6791,123-45-6792, expected:<0> but was:<2>
	at junit.framework.Assert.fail(Assert.java:57)
	at junit.framework.Assert.failNotEquals(Assert.java:329)
	at junit.framework.Assert.assertEquals(Assert.java:78)
	at junit.framework.Assert.assertEquals(Assert.java:234)
	at exe.Arthadb_Automation.ArthaDB_Customer_First_Last_Name_Check_TC011(Arthadb_Automation.java:450)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- ArthaDB_Customer_First_Last_Name_Check_TC011 -->
  <testcase name="ArthaDB_Customers_SSN_Check_TC012" time="0.018" classname="exe.Arthadb_Automation">
    <failure type="junit.framework.AssertionFailedError" message="CUSTOMERS- SSN check is Failed at CustomerPK=5, expected:&amp;lt;0&amp;gt; but was:&amp;lt;1&amp;gt;">
      <![CDATA[junit.framework.AssertionFailedError: CUSTOMERS- SSN check is Failed at CustomerPK=5, expected:<0> but was:<1>
	at junit.framework.Assert.fail(Assert.java:57)
	at junit.framework.Assert.failNotEquals(Assert.java:329)
	at junit.framework.Assert.assertEquals(Assert.java:78)
	at junit.framework.Assert.assertEquals(Assert.java:234)
	at exe.Arthadb_Automation.ArthaDB_Customers_SSN_Check_TC012(Arthadb_Automation.java:490)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- ArthaDB_Customers_SSN_Check_TC012 -->
  <testcase name="ArthaDB_PrimaryPhoneNumber_TC013" time="0.008" classname="exe.Arthadb_Automation">
    <failure type="junit.framework.AssertionFailedError" message="Customer - PrimaryPhoneNumber check is Failed at SSN=123-45-6789,123-45-6790, expected:&amp;lt;0&amp;gt; but was:&amp;lt;2&amp;gt;">
      <![CDATA[junit.framework.AssertionFailedError: Customer - PrimaryPhoneNumber check is Failed at SSN=123-45-6789,123-45-6790, expected:<0> but was:<2>
	at junit.framework.Assert.fail(Assert.java:57)
	at junit.framework.Assert.failNotEquals(Assert.java:329)
	at junit.framework.Assert.assertEquals(Assert.java:78)
	at junit.framework.Assert.assertEquals(Assert.java:234)
	at exe.Arthadb_Automation.ArthaDB_PrimaryPhoneNumber_TC013(Arthadb_Automation.java:526)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- ArthaDB_PrimaryPhoneNumber_TC013 -->
  <testcase name="ArthaDB_Customer_Details_Update_Check_TC014" time="0.017" classname="exe.Arthadb_Automation"/>
</testsuite> <!-- Test -->
